apps/vscode/command_client/command_client.py

```diff
@@ -123,26 +119,26 @@ def handle_existing_request_file(path):
         robust_unlink(path)
 
 
-def run_vscode_command(
+def run_command(
```

Comment:
while `run_vscode_command` is now too specific, `user.run_command` is a bit too generic. Maybe `editor_command`? `application_command`?